programs/witness_node/genesis.json

```diff
@@ -1,5 +1,5 @@
 {
-  "initial_timestamp": "2020-01-13T06:03:15",
```

Comment:
Remove this file from PR. Updated values are for test environment only, not for production. In general we use --create-genesis-json to recreate genesis file with latest values, and then finetune it to test values by scripts.